_data/simple-icons.json

```
@@ -2385,6 +2385,11 @@
             "hex": "3776AB",
             "source": "https://www.python.org/community/logos/"
         },
+        {
+            "title": "Qgis",
+            "hex": "589632",
+            "source": "https://www.qgis.org/"
```

For future reference it is easier of the source is the styleguide :wink: 

```suggestion
            "source": "https://www.qgis.org/en/site/getinvolved/styleguide.html"
```

---

Thanks for guiding, made the changes in latest commit. :)